core/util/proxy.py

```diff
@@ -0,0 +1,108 @@
+import requests
+from bs4 import BeautifulSoup
+import time
+
+class main:
+
+    def __init__(self, use=False, verbose=False):
+        '''
+        use : Determine whether to use proxy or not.
+        '''
+        self.use=use
+        self.verbose=verbose
+    iplist=[]
+    proxydict={}
+
+    def getproxy(self,use=False):
+        if self.use==True:
+            print("[PROXY] Gathering proxies. It's quite time consuming, so do some pushups in the meantime :P")
+            URL=['https://premproxy.com/list/']
+
+            i=2
+
+            while i<=8:
+                purl='https://premproxy.com/list/0'+str(i)+".htm"
+                URL.append(purl)
+                i+=1    
+
+            ips=[]
+            ips2=[]
+            purl=URL[0]
+
+            try:
+                for purl in URL:
+                    r=requests.get(purl)
+                    sp=BeautifulSoup(r.content,'html.parser')
+                    lst=list(sp.find_all('input',attrs={"name" : "proxyIp[]"}))
+                    for l in lst:
+                        ip=str(l.get('value'))[0:-6]
+                        ips.append(ip)
+                    time.sleep(2)
+                #print(ips[0:5])
+                print("[PROXY] Proxies gathered.")
+            except Exception as e1:
+                print(f"An error occured. Error details: {e1}")    
```

Comment:
self.framework.output for all. put a [PROXY] before the text.

Reply:
Done.